geode-docs/developing/data_serialization/PDX_Serialization_Features.html.md.erb

```diff
@@ -31,6 +31,25 @@ When you serialize an object using PDX, <%=vars.product_name%> stores the object
 
 This centralization of object type information is advantageous for client/server installations in which clients and servers are written in different languages. Clients pass registry information to servers automatically when they store a PDX serialized object. Clients can run queries and functions against the data in the servers without compatibility between server and the stored objects. One client can store data on the server to be retrieved by another client, with no requirements on the part of the server.
 
+**Note:**
+When pdx registry is not persistent or persistent pdx registry is removed, if all the servers are stopped/crashed with clients still running, then the client with previous pdx type info connects to cluster with servers that are re-started, the client will see Unknown pdx type error.
```

Comment:
Would there be any problem in servers apart from the clients getting errors?

Reply:
Not seen problems in sever side.